### PR TITLE
Sender address must be a f4 address for ETH transactions

### PR DESCRIPTION
**Pull request by aarshkshah1992**
Small update to the FIP to note that sender address must be an f4 address.

### CODE REVIEW

**Review by rvagg — APPROVED** [review]
sgtm, although is there a proper name for these other than `f4` since they're `t4` on calibnet (I think)? or is `f4` standard-enough language?

**Comment by jennijuju** [comment]
> sgtm, although is there a proper name for these other than `f4` since they're `t4` on calibnet (I think)? or is `f4` standard-enough language?

It’s  fine as it’s the name of an address class https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0048.md

File: FIPS/fip-0091.md [hunk 1]
```diff
@@ -126,6 +126,9 @@ Based on the above, the revised steps for processing and validating a Filecoin m
 4. Validate the transaction parameters and signature (after ignoring the first marker byte `0x02` for EIP-155 transactions as that is not part of the signature created by the user).
 5. Ensure that the `ChainID` signed by the user in the transaction digest is the same as the `ChainID` of the network
 
+Note that the sender address on the Filecoin message for both Homestead and EIP-155 transactions 
+must be an `f4` address or an ID address that can be resolved to a `f4` address. This behaviour is the same for existing EIP-1559 transactions.
```
Comment (TippyFlitsUK) [inline FIPS/fip-0091.md:130]:
```suggestion
must be an `f4` address or an ID address that can be resolved to an `f4` address. This behaviour is the same for existing EIP-1559 transactions.
```

**Review by TippyFlitsUK — APPROVED** [review]
Very small nit

**Comment by aarshkshah1992** [comment]
@jennijuju @TippyFlitsUK Please can one of you merge this for me ?

**Pull request merged**